src/index.js

```diff
@@ -21,6 +21,8 @@
  * for instance the href of a link, or a search term
  */
 export function sampleRUM(checkpoint, data = {}) {
+  sampleRUM.baseURL = sampleRUM.baseURL || window.rumBase || 'https://rum.hlx.page';
+  sampleRUM.baseURL = sampleRUM.baseURL.startsWith('/') ? `${window.location.origin}${sampleRUM.baseURL}` : sampleRUM.baseURL;
```

Comment:
this allows for relative paths which can include or not a  prefix, e.g. `/rum-proxy/` in case it is needed for customers which for instance are routing only part of their domain to AEM CS